### PR TITLE
update getVerificationCode

### PR DESCRIPTION
**Pull request by ibeckermayer**
Modifies the frontend and backend for the latest version of the VS, which allows us to use an api key to hit an admin server to issue tokens.

Also updates `testDate` to be `symptomDate` per the updated VS api.

### CODE REVIEW

File: config/functions.config.staging.js [hunk 1]
```diff
@@ -1,15 +1,13 @@
 var config = {
   client: {
-    url: "http://localhost:3000/",
+    url: "https://staging-permissions.covidwatch.org/",
```
Comment (colbymorrison) [inline config/functions.config.staging.js:3]: Should we also update this field in the prod file?

**Comment by colbymorrison** [comment]
Looks good so long as that axios request works correctly. I don't have the new verif server set up so I didn't test locally.

File: config/functions.config.staging.js (hunk 1 above)
Comment (jcolla-holla) [reply config/functions.config.staging.js:3]: Given this is the file `config/functions.config.staging.js` shouldn't this be our staging URL? permission-portal.covidwatch.org.  

Accordingly, shouldn't `config/functions.config.prod.js` have the prod URL: portal.covidwatch.org?

File: config/functions.config.local.js [hunk 2]
```diff
@@ -5,11 +5,9 @@ var config = {
   sendgrid: {
```
Comment (jcolla-holla) [inline config/functions.config.local.js:5]: right above here should update the URL to portal.covidwatch.org, right?  Or do we want to save that update for when we are ready to actually push this to prod and keep the scope of this PR more staging?  Not sure if matters either way.

**Pull request merged**